source/agora/api/Admin.d

```diff
@@ -25,6 +27,19 @@ public interface NodeControlAPI
 {
     @safe:
 
+    /***************************************************************************
+
+        Set the configuration of a Logger
+
+    ***************************************************************************/
+
+    public void postLogger (
+        @viaQuery("name") string name, @viaQuery("propagate") bool propagate = true,
```

Comment:
```suggestion
        @viaQuery("name") string name,
        @viaQuery("propagate") bool propagate = true,
```

Reply:
Where can I get information about `@viaQuery` in the `dlang` site ?

Reply:
It's a parameter UDA: https://vibed.org/api/vibe.web.common/viaQuery